gprofiler/profilers/registry.py

```diff
@@ -53,22 +51,26 @@ def register_profiler(
     default_mode="enabled",
     profiler_mode_argument_help: Optional[str] = None,
     profiler_arguments: Optional[List[ProfilerArgument]] = None,
+    disablement_help: Optional[str] = None,
 ):
+    # NOTE: Currently, the default_mode value is used as the default "enabled" mode in the default help string.
     if profiler_mode_argument_help is None:
         profiler_mode_argument_help = (
-            f"Choose the mode for profiling {profiler_name} processes. 'enabled'"
-            f" to profile them with the default method, or 'disabled' to disable {profiler_name} profiling"
+            f"Choose the mode for profiling {profiler_name} processes. '{default_mode}'"
+            f" to profile them with the default method, or 'disabled' to disable {profiler_name}-specific profiling"
         )
     if possible_modes is None:
```

Comment:
Any reason to support this case? No profiler should use it

Reply:
Are you sure we don't ever want to enable profilers having just `enabled` and `disabled` options?

Reply:
Nah, let's be explicit and consistent and always call optimizers by their name